### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by morchickit**
Adding about info about the user warning.

### CODE REVIEW

**Comment by BobHarper1** [comment]
I think we could be more general about what might appear. I got another from the openpyxl package `UserWarning: Unknown extension is not supported and will be removed`. Basically, get.py will run through the program, some errors may occur but can usually be ignored, and you can review how it's turned out in data/report.csv

**Comment by morchickit** [comment]
I change it. Have a look now @BobHarper1

**Comment by BobHarper1** [comment]
I made a couple changes to your commit. If @robredpath is happy I think that's good to go.

File: README.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ Run:
 # Warning this deletes previously downloaded data
 mkdir -p data/{original,json_all,json_valid,json_acceptable_license,json_acceptable_license_valid}
 python get.py
+
+You might see different user warnings from python while running the script. You can happily ignore most of those that don't cause the program to fail. Run data/report.csv to review what datasets have been downloaded.
```
Comment (robredpath) [inline README.md]: Can we change the first sentence to something like: "When running the datagetter, you may see several UserWarnings." ? I think that'll make it clearer what a UserWarning is.

Comment (BobHarper1) [reply README.md]: Yep, updated

**Pull request merged**